pre-docs/writing_plans.md

```diff
@@ -510,3 +510,7 @@ plan pdb_discover {
 }
 ```
 
+## Converting YAML Plans to Puppet Plans
```

Comment:
I'm hesitant to create a whole section that consists of just a link, because it makes for weird UX. Could we do one of the following instead?

* leave this section out entirely, with or without a See also link in this page's top-level overview to the "Converting YAML plans to Puppet plans" topic on the writing_yaml_plans page
* re-use/duplicate the same content from writing_yaml_plans.md

Reply:
A "See also" link seems like the right fit - I'll remove the section, is there anything I should do to add a "see also"?

Reply:
If you could just add the following at line 8/9 of this file:

Related information:
Converting YAML plans to Puppet plans